src/herbivoresAndCarnivores.js

```diff
@@ -2,14 +2,42 @@
 
 class Animal {
   // write your code here
+  health = 100;
```

Comment:
I think health would be better pass as a parameter to constructor with default value, for some case.

Reply:
if you ask me, its redundant, becouse its weel be identically

Reply:
ok)